config/config.exs

```diff
@@ -10,6 +10,8 @@ import Config
 config :lightning,
   ecto_repos: [Lightning.Repo]
 
+config :lightning, Lightning.Repo, types: Lightning.PostgrexTypes
```

Comment:
@wanecode do we still need this since you used `nullif`?

Reply:
@stuartc. Yes . the max clause needs infinity. Or we can hard a "max date"